depends/common/2048/CMakeLists.txt

```diff
@@ -23,8 +23,13 @@ elseif(CORE_SYSTEM_NAME STREQUAL linux)
 elseif(CORE_SYSTEM_NAME STREQUAL osx)
   set(BUILD_COMMAND $(MAKE) -C . -f Makefile.libretro  platform=osx ${LIBRETRO_DEBUG} GIT_VERSION=)
 elseif(CORE_SYSTEM_NAME STREQUAL ios OR CORE_SYSTEM_NAME STREQUAL darwin_embedded)
+  if(CORE_PLATFORM_NAME STREQUAL ios)
```

Comment:
This change should be PRd to https://github.com/kodi-game/kodi-game-scripting/

Reply:
sure, will do.